cost-analyzer/templates/cost-analyzer-oidc-config-map-template.yaml

```diff
@@ -24,6 +24,7 @@ data:
         "discoveryURL" : "{{ .Values.oidc.discoveryURL }}",
         "hostedDomain" : "{{ .Values.oidc.hostedDomain }}",
         "skipOnlineTokenValidation" : "{{ .Values.oidc.skipOnlineTokenValidation | default "false" }}",
+        "useClientSecretPost": {{ .Values.oidc.useClientSecretPost }},
```

Comment:
@kaelanspatel Just out of curiosity here - would this line need a ` | default "false" ` just like the line above it?